test/integration/patients/worklist/bulk-edit.js

```diff
@@ -31,10 +31,13 @@ context('Worklist bulk editing', function() {
 
         fx.data[0].relationships.patient = { data: { id: '1' } };
         fx.data[0].relationships.state = { data: { id: '22222' } };
+        fx.data[0].attributes.created_at = testTs();
         fx.data[1].relationships.patient = { data: { id: '2' } };
         fx.data[1].relationships.state = { data: { id: '22222' } };
+        fx.data[1].attributes.created_at = testTsSubtract(1);
         fx.data[2].relationships.patient = { data: { id: '3' } };
         fx.data[2].relationships.state = { data: { id: '22222' } };
+        fx.data[2].attributes.created_at = testTsSubtract(2);
 
```

Comment:
This test was getting a worklist item in the list via the patient name and then using `.prev()` to get the checkbox element. But that breaks the test because the `.prev()` element is now the patient sidebar button.

So, I added test dates to each item in the worklist so they are forced to be in the same order each time.

And then updated the test to retrieve each row item by their index position (e.g. `.first()`, `.eq(1)`, `.last()`) in the list, instead of using the patient name (e.g. `.contains('Patient One')`)